vignettes/teal-data-reproducibility.Rmd

```diff
@@ -10,34 +12,50 @@ vignette: >
 
 # Reproducibility of `teal_data` objects
 
-Reproducibility is a primary function of the `qenv` class, which `teal_data` inherits from. Every data modification in `teal_data` object is performed in an encapsulated environment, separate from the global environment.
+Reproducibility is a primary function of the `qenv` class, which `teal_data` inherits from.
+Every data modification in a `teal_data` object is performed in an encapsulated environment, separate from the global environment.
 
-It is important to note that the reproducibility of this object is limited only to the data-code relationship. Other aspects such as the reliability of the data source, reproducibility of the R session (including package versions), and creation and use of objects from other environments (e.g. `.GlobalEnvironment`) cannot be verified properly by the `teal_data`.
+It is important to note that the reproducibility of this object is limited only to the data-code relationship.
+Other aspects such as the reliability of the data source, reproducibility of the R session (including package versions), and creation and use of objects from other environments (e.g. `.GlobalEnv`) cannot be verified properly by `teal_data`.
+It is advisable to always begin analysis in a new session and run all code that pertains to the analysis within the `teal_data` object.
 
-## Verification status
+## Verification
+
+### Verification status
+
+Every `teal_data` object has a _verification status_, which is a statement of whether the contents of the `env` can be reproduced by `code`.
+From this perspective, `teal_data` objects that are instantiated empty are _verified_ but ones instantiated with data and code are _unverified_ because the code need not be reproducible.
```

Comment:
Very minor
```suggestion
From this perspective, `teal_data` objects that are instantiated empty are _verified_ but ones instantiated with data and code parameters are _unverified_ because the code need not be reproducible.
```

Reply:
`... because the code was not executed and might not be reproducible`?
`... because the code was not executed and might not be able to reproduce objects specified in data`?

Reply:
:thinking: There is an implicit assumption that we either evaluate code or we create an object with code. Do you think we should explicitly state that in the latter case the code is not run?

Reply:
Maybe, but just maybe! someone can assume that code during `teal_data(code = )` creation is executed :)
Only for those I would propose to state this was not executed. But if this is stated in Getting Started / Introduciton then we can avoid it here and leave it as it is.

Reply:
I'm on the fence on this one. Perhaps we should add this information to the `teal_data` help file?

Reply:
Yes, let's add a one liner and you can then omit this suggestion